docs/topics/about-home-var.adoc

```diff
@@ -16,4 +16,4 @@ If you are installing on a Windows operating system:
 . If a *Confirm file replace* window is displayed during extraction, click *Yes to all*.
 ====
 
-When you encounter `<MTA_HOME>` in this guide, replace it with the actual path to your {ProductShortName} installation.
```

Comment:
Are you sure that these modules will never be used for MTA? I think the replacements should be done as doc attributes unless there is a very good reason not to do so.

Reply:
@apinnick Using doc attributes is what we plan to do -- so please confirm -- this should be <{ProductShortName}_HOME> with a way to determine if it's MTA or MTR per document, correct? (that method will involve a variable with values for the variable for MTA and MTR)